components/kubeflow/kfserving/Dockerfile

```diff
@@ -9,5 +9,5 @@ ENV APP_HOME /app
 COPY src $APP_HOME
 WORKDIR $APP_HOME
 
-ENTRYPOINT ["python"]
+ENTRYPOINT ["python3"]
```

Comment:
if we are on a python3 image. do we explicitly need to say python3?

Reply:
Hah, guess not. I can keep it as `python`